security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,28 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-3-0"]
+== OpenShift Compliance Operator 1.3.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.3.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:5102[RHBA-2023:5102 - OpenShift Compliance Operator enhancement update]
+
+[id="compliance-operator-1-3-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* Compliance Operator 1.3.0 now supports creation of the DISA-STIG profile within the supported profiles module for {product-title} 4.14. See https://docs.openshift.com/container-platform/4.13/security/compliance_operator/compliance-operator-supported-profiles.html#compliance-supported-profiles_compliance-operator-supported-profiles
+ for more information.
+
+* Compliance Operator 1.3.0 now provide support to IBM Power and IBM Z for NIST moderate profiles. 
```

Comment:
Couple things: we have IBM architecture attributes you can use. I also recommend using the full title for the profiles, of which there are two: platform and node.

"Compliance Operator 1.3.0 now supports :ibmpowerProductName: and :ibmzProductName: for NIST 800-53 Moderate-Impact Baseline for Red Hat OpenShift platform and node profiles. "

Reply:
Fixed, thank you!